ts/Stock/StockToolsBindings.ts

```diff
@@ -1994,6 +1994,8 @@ var stockToolsBindings: Highcharts.Dictionary<Highcharts.NavigationBindingsOptio
     /**
      * Displays chart in fullscreen.
      *
+     * Fullscreen is not supported on iPhone due to iOS limits.
```

Comment:
Could you use bold `**Note**:` at the beginning? To highlight this :)

Reply:
Fixed